src/Docker.DotNet/Endpoints/ContainerOperations.cs

```diff
@@ -76,7 +75,6 @@ public async Task<ContainerProcessesResponse> ListProcessesAsync(string id, Cont
             {
                 throw new ArgumentNullException(nameof(id));
             }
-
```

Comment:
Why remove these lines?

Reply:
Should have been restored now

Reply:
Same as https://github.com/dotnet/Docker.DotNet/pull/480#discussion_r552341101

Reply:
See https://github.com/dotnet/Docker.DotNet/pull/480#discussion_r552341101